content/docs/tutorials/get-started/versioning-basics.md

```diff
@@ -0,0 +1,250 @@
+
+</details>
+
+Refer to
+[Versioning Data and Model Files](/doc/use-cases/versioning-data-and-model-files)
+and `dvc add` for more information on versioning data with DVC.
+
+## Store and share data
+
+Now that your raw data is tracked by DVC, you can push it from your repository
+to the default [remote storage](/doc/command-reference/remote).
+
+> As seen in the intro's [Configure](/doc/tutorials/get-started#configure)
+> section, we are using a _local remote_ for illustrative purposes.
+
+```dvc
+$ dvc push
+```
+
+Similar to pushing source code to a _Git remote_, `dvc push` ensures that your
+data files and models are safely backed up remotely. This means that the data
+can be pulled by yourself or by colleagues when and where needed. Usually, we
+also want to `git commit` and `git push`, to save the new (or changed versions
+of) [DVC-files](/doc/user-guide/dvc-file-format).
+
+<details>
+
+### Expand to learn more about DVC internals
+
+You can check that the data has been backed up to the remote (`/tmp/dvc-storage`
+local directory) with:
+
+```dvc
+$ ls -R /tmp/dvc-storage
+...
+/tmp/dvc-storage/a3:
+04afb96060aad90176268345e10355
+```
+
+</details>
+
+## Retrieve data
+
+Imagine you're just cloning the Git repo that has been created so far in another
+computer. This can be simulated by cloning our **example-get-started** repo from
+Github, and checking out the
+[`3-add-file`](https://github.com/iterative/example-get-started/tree/3-add-file)
+tag:
+
+```dvc
+$ cd ~
+$ git clone https://github.com/iterative/example-get-started
+$ cd example-get-started
+$ git checkout 3-add-file
+```
+
+If you list the files in this fresh <abbr>workspace</abbr>, or even in the
+cache, you'll notice that the `data/data.xml` file is not there yet. This is
+because it's not stored by Git! To get it, simply run:
+
+```dvc
+$ dvc pull
+```
+
+`dvc pull` downloads data files that are referenced in all present
+[DVC-files](/doc/user-guide/dvc-file-format) from the <abbr>project</abbr>'s
+remote storage, so usually we run it after `git clone`, `git pull`, or
+`git checkout`.
+
+Alternatively, if you want to retrieve a single file or directory, you can
+specify the target like this:
+
+```dvc
+$ dvc pull data/data.xml.dvc
+```
+
+> In this case, both commands have the same result, as there's currently just
+> one DVC-tracked file in the repo.
+
+[DVC remotes](/doc/command-reference/remote), `dvc push`, and `dvc pull` provide
+a basic collaboration workflow, the same way as Git remotes, `git push` and
+`git pull`. See
+[Sharing Data and Model Files](/doc/use-cases/sharing-data-and-model-files) for
+more information.
+
+Feel fee to remove the cloned example repo for now:
+
+```dvc
+$ cd ..
+$ rm -rf example-get-started
+```
+
+## Import data
+
+We've seen how to [push](#store-and-share-date) and [pull](#retrieve-data) data
+from/to a remote storage. But what if we wanted to integrate a dataset or ML
+model produced in one project into another one?
```

Comment:
> Import data - should become something like "Accessing Data" and mention stuff like get, api, etc. 

I'm not sure we want to complicate it that much in the Get Started. `dvc get` is previously explained in a note but the API seems like too much, definitely not part of basic versioning... I just added a note with a link to that ref. for now.

> We need to connect them somehow with the example-get-started in a meaningful way

I already included an import stage in the actual repo but yeah, it's not super meaningful. It just replaces the previous get+add steps with import. BUT actually it causes a problem later, see https://github.com/iterative/dvc.org/pull/1074#pullrequestreview-393355450

Reply:
> I'm not sure we want to complicate it that much in the Get Started.

if it becomes a comlication, most likely it's a sign that we need to take a lighter approach in general? Like we discussed, the thing we want to avoid is building one mode Deep Dive tutorials. If needed we can hide all commands that are required to reproduce this. All the long explanations, etc.

Reply:
Ivan's reply:

> if we see "Versioning Basics" as "data management" (on a conceptual level): with DVC we are enforcing some way of storing data. And to me it's important to explain "Data access" as well as the management.
It's a good question how to name this [vs "Import data"] - but it should be about list, get, import, dvc.api

My take is you're right, but it's starting to feel like we need another section of Get Started just for this. So I'm going to leave this for later...

Reply:
💡 Maybe merge versioning-basics#store-and-share-data + versioning-basics#retrieve-data (push and fetch/pull in a single section);

And then make versioning-basics#import-data -> data-versioning#data-retrieval (list, import, dvc.api)
> get is already used and explained in this page's intro but we can mention it again.

Reply:
> merge ... push and fetch/pull in a single section

Done